cfme/infrastructure/virtual_machines.py

```diff
@@ -1307,3 +1313,15 @@ class VmEngineRelationship(CFMENavigateStep):
     def step(self):
         self.prerequisite_view.toolbar.configuration.item_select(
             'Edit Management Engine Relationship')
+
+
+@navigator.register(Vm, 'Explore')
+class VmExplore(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
+    VIEW = InfraVmDetailsView
+
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.navigation.select('Compute', 'Infrastructure', '/vm_infra/explorer')
```

Comment:
shouldn't here be an item name from menu instead of this id ?

Reply:
The View that this class is returning is different from the 'All' navmazing class.